dashboard/app/helpers/locale_helper.rb

```diff
@@ -34,8 +34,6 @@ def options_for_locale_select
     options
   end
 
-  private
```

Comment:
Instead of removal, should this be replaced (repeatedly) by `private_class_method`?